debian/copyright

```diff
@@ -0,0 +1,72 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Contact: Christopher Rosell <chrippa@tanuki.se>
+Source: https://github.com/streamlink/streamlink
+Upstream-Name: streamlink
+
+Files: *
+Copyright: 2011-2016 Christopher Rosell <chrippa@tanuki.se>
```

Comment:
Another copyright line needs to be added here in addition to this one. Is 2011-2016 correct? Or is it 2011-2015?